draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1933,6 +1950,11 @@ Additionally, client-side techniques such as those described in
 {{app-isolation}} may also prove effective against CSRF, and are certainly worth
 exploring in combination with "SameSite" cookies.
 
+#### Taking Scheme into Consideration
+By using the scheme, as well as the registrable domain, SameSite can help to
+protect a secure site against a network attacker which is impersonating an
+insecure site with the same registrable domain.
```

Comment:
I think it's worth spelling out http vs https specifically. Perhaps: [...], "SameSite" cookies can help to protect https origins against a network attacker that is impersonating a corresponding http origin.

(I believe this should be "that" rather than "which"?)

Reply:
SGTM